docs/workflows/index.rst

```diff
@@ -28,6 +28,4 @@ set is the hostname and port::
 .. toctree::
    :maxdepth: 2
 
-   sync
-   upload
-   publish-host
+   cache
```

Comment:
why did you keep only cache use case?

Reply:
Because there is only one workflow right now. It's the cache one.